0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/json4s/CommonSerializers.scala

```diff
@@ -109,4 +109,17 @@ object CommonSerializers {
       case OneOf(schemas) => JArray(schemas.map(Schema.normalize(_)))
     }
     ))
+
+  object AnyOfSerializer extends CustomSerializer[AnyOf](_ => (
+    {
+      case JArray(values) =>
+        val schemas: List[Option[Schema]] = values.map(Schema.parse(_))
+        if (schemas.forall(_.isDefined)) AnyOf(schemas.map(_.get))
```

Comment:
I feel guilty that I used this smelly approach in `OneOfSerializer`, but virtually we should never use `Option.get` (it is always possible to avoid it). This could be rewritten as:

```scala
schemas.sequence match {
  case Some(s) => AnyOf(s)
  case None => throw new ...
```

Maybe couple of cats imports also will be necessary.